srcpkgs/emilua/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'emilua'
+pkgname=emilua
+version=0.4.3
+revision=1
+archs="x86_64"
+build_style=meson
+build_wrksrc="${pkgname}-v${version}"
+hostmakedepends="LuaJIT-devel boost-devel fmt-devel openssl-devel ncurses-devel serd-devel sord-devel liburing-devel libcap-devel"
+makedepends="meson boost re2c gawk gperf xxd ruby-asciidoctor pkg-config"
+short_desc="Lua execution engine"
+maintainer="Valter Nazianzeno <manipuladordedados@gmail.com>"
+license="BSL-1.0"
+homepage="https://gitlab.com/emilua/emilua"
+distfiles="https://github.com/BoostGSoC14/boost.http/archive/93ae527c89ffc517862e1f5f54c8a257278f1195.tar.gz
+https://github.com/breese/trial.protocol/archive/79149f604a49b8dfec57857ca28aaf508069b669.tar.gz
+https://gitlab.com/emilua/emilua/-/archive/v${version}/emilua-v${version}.tar.gz"
+checksum="4e90417806c285348d968183072918e86d7fd7d7db5affb63ec03faee92ed8ed b45bc998c30e6226a72cbe37863793bb2d8f6160918bc441a469d3e405dd4353
+c70033f50a067651943b15cd5fed0d407ab79bc2c686c68af796ace981a746ae"
+
+configure_args="
+	-Dversion_suffix=-void1 \
+	-Denable_http=true \
+	-Denable_file_io=true \
+	-Denable_io_uring=true \
+	-Denable_linux_namespaces=true \
+	-Denable_tests=true \
+	-Denable_manpages=true"
```

Comment:
Indent with a single space and remove the trailing backslashes, they are not necessary.